transforming-message-formats/main.bal

```diff
@@ -8,35 +8,36 @@ final http:Client hospitalServiceEP = check initializeHttpClient();
 
 function initializeHttpClient() returns http:Client|error => new (hospitalServiceUrl);
 
-type HealthcareReservation record {|
-    string name;
+type HealthcareReservation record {
+    string firstName;
+    string lastName;
     string dob;
-    string ssn;
+    int[3] ssn;
     string address;
     string phone;
     string email;
     string doctor;
-    string hospital_id;
+    string hospitalId;
     string hospital;
-    string card_no;
-    string appointment_date;
-|};
+    string cardNo;
+    string appointmentDate;
+};
 
-type Patient record {|
+type Patient record {
     string name;
     string dob;
     string ssn;
     string address;
     string phone;
     string email;
-|};
+};
 
-type HospitalReservation record {|
+type HospitalReservation record {
```

Comment:
Can we make these closed?

Reply:
These are generated records. However, We can make this change to the source code